ftw/simplelayout/utils.py

```diff
@@ -14,3 +19,14 @@ def get_block_html(block):
                                    IBlockProperties)
     viewname = properties.get_current_view_name()
     return block.restrictedTraverse(viewname)()
+
+
+def get_block_types():
+    platform = getSite()
+    types_tool = getToolByName(platform, 'portal_types')
+
+    dexterity_ftis = filter(
+        IDexterityFTI.providedBy, types_tool.objectValues())
+
+    return filter(
```

Comment:
This returns the FTI-object, but I think it should return the ID of the FTI, right=